.github/build.yaml.gomplate

```diff
@@ -300,10 +300,7 @@
         with:
           name: build-{{{ $flavor }}}-{{{ $config.arch }}}
           path: build
-      - name: Add build repository
-        run: |
-          export DOCKER_INSTALL=true
-          sudo -E make add_local_repo
+      {{{ tmpl.Exec  "make" "add_local_repo" }}}
```

Comment:
I am still tempted to add here an additional step to upgrade repositories so we use the built artifacts to build the iso. I know this will break cross arch ISO builds, however I am wondering if it wouldn't be easier to not run cross arch builds and relay on pure arch specific CI pipes?

Reply:
The problem with pure arch specific pipes is that we then add extra jobs for anything that requires arm64 and we only have one worker ready, which means that we increase the time for arm64 jobs. Although currently there is not that much movement on arm64 so it may be easier to just dont do cross arch anymore and avoid this stupid problem of deps which may be more important.

Anyway, this is just a speed VS deps issue. I would go with fixing the deps issue, which is more problematic than having to way half an hour more IMO. If in the future we get more heavy development in the toolkit and it requires more arm64 specific development, we can also temporarily increase the arm64 workers, while we cannot fix the dep problems easily.

Reply:
Ok just revisited and I think that adding the proper built arch in `add_local_repo` should do the trick. The problem is that the luet interface does not allow to set repo architecture form the command line, hence the yaml file for the repository should be modified manually. I guess should test this approach by a manual hack on the `add_local_repo`  target and if it works then code the correspondent arch flag in `luet add repo` command.